inspirehep/modules/records/api.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2016 CERN.
+#
+# Invenio is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+""""Inspire Records"""
+
+from __future__ import absolute_import, division, print_function
+
+from datetime import datetime
+
+import arrow
+from elasticsearch.exceptions import NotFoundError
+
+from invenio_pidstore.errors import PIDDoesNotExistError
+from invenio_records.api import Record
+
+from inspirehep.utils.record_getter import (
+    RecordGetterError,
+    get_es_record_by_uuid
+)
+from inspirehep.utils.record import (
+    merge_pidstores_of_two_merged_records,
+    soft_delete_pidstore_for_record,
+)
+
+
+class InspireRecord(Record):
+
+    """Inspire Record inherits from Invenio Record"""
+
+    def merge(self, new):
+        merge_pidstores_of_two_merged_records(new.id, self.id)
+
+    def delete(self, force=False):
```

Comment:
Needs a docstring. Also, do we really need the `force` parameter? We're subclassing to get rid of the default behavior, because we need something different from this method, so what is the rationale for allowing it? Maybe @kaplun or @jmartinm know?